aries-site/src/examples/templates/forms/SignUpExample.js

```diff
@@ -92,6 +93,11 @@ export const SignUpExample = () => {
       const adjustedRule = { ...rule };
       const valid = adjustedRule.regexp.test(values.password);
       adjustedRule.valid = valid;
+      if (valid !== true) {
+        setBackgroundError('#FC61613D');
+      } else {
+        setBackgroundError('white');
```

Comment:
Should this and the default backgroundError be `undefined`? I assume when I switch to dark mode, we wouldn't want white to be there.

Reply:
yes thanks for catching!